searx/settings.yml

```diff
@@ -72,13 +72,13 @@ server:
   bind_address: "127.0.0.1"
```

Comment:
please revert modifications not needed by this patch.

Reply:
Done in [02bf446](https://github.com/searxng/searxng/pull/2514/commits/02bf4469c0db0b0d856b88d7a632646d147d275d)